dbc.dd

```diff
@@ -181,6 +181,13 @@ class Date
 	$(LI postconditions)
 	)
 
+    $(P Because at the start of class/struct destructor object instance may
+    point to other object instances already collected by the GC trying to
```

Comment:
This needs a comma after "GC" so it doesn't parse as "... instances collected by [the GC trying to check...]".